guardian/support-frontend
# Refactor payment success metric creation

## What are you doing in this PR?

Refactoring the creation of `Metric`s in the support-workers CDK, in preparation for adding a new alarm for Tier Three.



[**Trello Card**](https://trello.com/c/Pf7IkuIx/1271-tier-3-no-acquisitions-alarm)

## Why are you doing this?

This code is all very similar, and I'm about to add another alarm for Tier 3.



## How to test

Run the snapshot tests to verify nothing has changed (this is purely a refactoring, the generated Cloudformation template is identical):

```
$ yarn test
```



## How can we measure success?



## Have we considered potential risks?



## Accessibility test checklist


## Screenshots
